tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -147,7 +147,9 @@ public function setUp()
         static::setAppNamespace('Cake\Test\TestCase\View\Helper');
         $this->View = new View();
 
-        $this->Form = new FormHelper($this->View);
+        $this->Form = new FormHelper($this->View, [
+            'autoSetCustomValidity' => false,
+        ]);
```

Comment:
Should we have a few more tests that do cover this feature if we're going to enable it for all users?

Reply:
Sure, what other tests would you like to see? I disabled it here because updating all the existing tests seemed tedious, but I can do that too if you like.

Reply:
I feel like there should be a test that covers each input type and some associations.